### PR TITLE
Set up the Document table in entities

### PR DESCRIPTION
**Pull request by vtruo009**

### CODE REVIEW

File: src/server/src/entities/documents.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @Column({ nullable: true })
+    isDefault: boolean;
+
+    @Column({ nullable: false})
+    dateAdded: Date;
+
+    @Column({ type: 'bytea', nullable: false })
+    document: Buffer;
+
+    @ManyToOne(() => Student, (student) => (student.documents))
```
Comment (johan1505) [inline src/server/src/entities/documents.ts]: The student's class has no documents attribute. You need to specify the relationship in the student entity as well.

**Review by johan1505 — CHANGES_REQUESTED** [review]

File: src/server/src/entities/documents.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import {
+    Entity,
+    Column,
+    PrimaryGeneratedColumn,
+    ManyToOne,
+    BaseEntity,
+} from 'typeorm';
+import { Student } from '../entities/student';
```
Comment (johan1505) [inline src/server/src/entities/documents.ts]: I think you can just do ./student in the import to get the Student class

File: src/server/src/entities/documents.ts (hunk 1 above)
Comment (vtruo009) [reply src/server/src/entities/documents.ts:27]: I thought you said the online version has the documents attribute or we will have the documents attribute in the Student class?

File: src/server/src/entities/documents.ts (hunk 2 above)
Comment (vtruo009) [reply src/server/src/entities/documents.ts:8]: that gave me an error though, saying Student class has no default export. Do I add the export to Student class?

File: src/server/src/entities/documents.ts (hunk 1 above)
Comment (vtruo009) [reply src/server/src/entities/documents.ts]: Sorry forgot to add that part. Added now

File: src/server/src/entities/student.ts [hunk 3]
```diff
@@ -58,4 +59,7 @@ export class Student extends BaseEntity {
 
     @OneToMany(() => JobApplication, (jobApplication) => jobApplication.student)
     public jobApplications: JobApplication[];
+
+    @OneToMany(() => Document, (document) => document.student)
+    public document: Document[];
```
Comment (johan1505) [inline src/server/src/entities/student.ts]: documents

File: src/server/src/entities/document.ts [hunk 4]
```diff
@@ -24,6 +24,6 @@ export class Document extends BaseEntity {
     @Column({ type: 'bytea', nullable: false })
     document: Buffer;
 
-    @ManyToOne(() => Student, (student) => (student.documents))
+    @ManyToOne(() => Student, (student) => (student.document))
```
Comment (johan1505) [inline src/server/src/entities/document.ts]: student.documents

**Review by johan1505 — CHANGES_REQUESTED** [review]

**Review by johan1505 — APPROVED** [review]

**Pull request merged**